<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analizador de Archivos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
   QMainWindow {
       background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                                         stop:0 #1a2a6c, stop:1 #b21f1f);
   }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>951</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
     QLabel {
         color: #333333;
         background-color: rgba(255, 255, 255, 0.9);
         border-radius: 15px;
         font-size: 24px;
         font-weight: bold;
         padding: 20px;
         qproperty-alignment: AlignCenter;
         border: 2px solid #ffffff;
     }
     </string>
    </property>
    <property name="text">
     <string>NINGÚN ARCHIVO CARGADO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agregarButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
     QPushButton {
         background-color: #4CAF50;
         color: white;
         border: none;
         border-radius: 25px;
         font-size: 16px;
         font-weight: bold;
         padding: 10px 20px;
     }
     QPushButton:hover {
         background-color: #45a049;
     }
     QPushButton:pressed {
         background-color: #3e8e41;
     }
     </string>
    </property>
    <property name="text">
     <string>Seleccionar archivo</string>
    </property>
   </widget>
  </widget>
  <action name="actionGraficos">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Generar Gráfico</string>
   </property>
  </action>
  <action name="actionfuncionabilidadPrueba">
   <property name="text">
    <string>Análisis Avanzado</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
